src/shared/components/card-2/card-2.css

```diff
@@ -0,0 +1,83 @@
+/**
+ * Block
+ */
+
+.hw-card-2 {
+  position: relative;
+  background: var(--hw-color-white);
+  transition: transform 0.3s;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: var(--hw-spacing);
+
+  @media(--max-medium){
+    flex-direction: row;
+    min-height: 150px;
+
+    &__media {
+      width: 60%;
+      min-width: 120px;
+    }
+  }
+
+  &::after {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    transition: opacity 1s cubic-bezier(0.165, 0.84, 0.44, 1);
+    background: var(--hw-color-opaque-black-7);
+    content: '';
+    opacity: 0;
+  }
+
+  &:hover {
+    &::after {
+      opacity: 1;
+    }
+  }
+  &:hover .hw-card-2__body {
+    background-color: var(--hw-color-gray-lighter);
+    border: 3px solid #C1C1C1;
+  }
+
+  /**
+   * Elements
+   */
+
+  &__media {
+    object-fit: cover;
+    overflow: hidden;
+    height: 190px;
+    flex-grow: 1;
+  }
```

Comment:
`__media-inner` should probably have height 100%. As it is now, the image does not show.